### PR TITLE
First (colliding)mapping done with the new tool

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Review by BleachDrinker420 — COMMENTED** [review]
Looks good except these two which could be changed to match upstream and keep the "shape" name consistent

**Comment by u9g** [comment]
Changed those to match

**Review by BleachDrinker420 — APPROVED** [review]

File: mappings/net/minecraft/world/BlockView.mapping [hunk 1]
```diff
@@ -10,6 +10,7 @@ CLASS net/minecraft/class_1158 net/minecraft/world/BlockView
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD method_3774 getBlockAt (III)Lnet/minecraft/class_197;
```
Comment (BoogieMonster1O1) [inline mappings/net/minecraft/world/BlockView.mapping:13]: shouldn't this be `getBlockState`? unless it returns a Block in which case it should be `getBlock`

File: mappings/net/minecraft/block/Block.mapping [hunk 2]
```diff
@@ -645,12 +710,41 @@ CLASS net/minecraft/class_197 net/minecraft/block/Block
 			COMMENT The block's strength
 	METHOD method_443 (Lnet/minecraft/class_236;)Z
 		ARG 1 vec
+	METHOD method_444 getStrongPower (Lnet/minecraft/class_1158;IIII)I
```
[the remark this answers is not in the document]
Comment (u9g) [reply mappings/net/minecraft/block/Block.mapping]: done in 6f7fe09

File: mappings/net/minecraft/world/BlockView.mapping (hunk 1 above)
Comment (BleachDrinker420) [reply mappings/net/minecraft/world/BlockView.mapping]: Pretty sure that method is for versions before BlockState existed

**Review by arthurbambou — APPROVED** [review]

**Pull request merged**